app/src/modules/tageuro/tageuro.js

```diff
@@ -0,0 +1,191 @@
+                            apiRes.fromCache = false;
+                            resolve(apiRes);
+                        }).catch((error) => {
+                            reject(error);
+                        });
+                            
+                    }
+                });
+            } else {
+                reject("Couldn't process query.")
+            }
+        });
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your name needs to be translated, use this function getName().
+     * @returns the tab name translated
+     */
+
+    getName: function (i18n) {
+        const _ = i18n._;
+        return _("TagEuro", "tageuro");
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * Otherwise, if your name doesn't need to be translated, use this attribute.
+     */
+
+    name: "TagEuro",
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your keyword needs to be translated, use this function getKeyword().
+     * @returns keyword translated
+     */
+    getKeyword: function (i18n) {
+        const _ = i18n._;
+        var finalKeywords = ""
+        keywords.forEach(function(element) {
+            finalKeywords += element + "|"
+        });
+
+        finalKeywords = finalKeywords.substr(0, oldStr.length-1);
+        return _("Nicolas Dupont(\-| )Aignan", "tageuro");//finalKeywords;
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * Otherwise, if your keyword doesn't need to be translated, use this attribute.
+     * The keyword can be a regex. If you need help for your regex, use this https://regex101.com/#javascript
+     */
+
+    keyword: "(N|n)icolas (D|d)upont(\-| )(A|a)ignan|(F|f)lorian (P|p)hilippot|(R|r)apha(\ë|e)l (G|g)lucksmann|(J|j)ean (C|c)hristophe (L|l)agarde|(F|f)ran(\ç|c)ois (A|a)sselineau|(I|i)an (B|b)rossat|(J|j)ean (L|l)assalle|(Y|y)annick (J|j)adot|(F|f)ran(\ç|c)ois (X|x)avier (B|b)ellamy|(N|n)athalie (L|l)oiseau|(J|j)ordan (B|b)ardella|(M|m)anon (A|a)ubry|(N|n)athalie (A|a)rthaud",
+    //keyword: "Nicolas Dupont(\-| )Aignan",
+    /**
+     * (OPTIONAL)
+     * script : If your IA includes a script, place it under public/javascript/xxx.js and replace "hello" by "xxx".
+     */
+
+    //script: "js",
+
+    /**
+     * (NEEDED)
+     * triggers : Depending on the trigger, the keyword needs to be placed at a specific point in the query.
+     * It has 4 different values :
+     * 			start  : keyword + string
+     *          end    : string + keyword
+     *          any    : string + keyword + string
+     *          strict : perfect match with keyword
+     */
+
+    trigger: "strict",
+
+    /**
+     * (NEEDED)
+     * flag : Only 3 flags allowed : (default : i)
+     * 			- g : global
+     * 			- m : multi-line
+     * 			- i : insensitive
+     */
+
+    flag: "gim",
+
+    /**
+     * (NEEDED)
+     * timeout : Time before your response is considered as canceled (in milliseconds)
+     */
+
+    timeout: 5000,
```

Comment:
We try not to exceed 3 seconds of timeout to get fast results
```suggestion
    timeout: 3000,
```